kpops/utils/gen_schema.py

```diff
@@ -1,3 +1,5 @@
+# type: ignore[reportGeneralTypeIssues]
```

Comment:
pyright is breaking on CI: https://github.com/bakdata/kpops/actions/runs/12412577169/job/34652661630?pr=571

It passes locally on my machine.

Reply:
is that after you reverted poetry.lock?

Reply:
Yes. :/

Reply:
but why? you didn't make any changes to this file

Reply:
I have no clue why. I couldn't find a good answer. The errors are also very generic, and I don't really understand why they are being triggered.

Reply:
I think you need to revert poetry.lock to `main` and run `poetry lock --no-update`

Reply:
The poetry.lock file keeps being overriten. Even if I delete the `.venv` folder and revert the `poetry.lock` file and run `poetry lock --no-update`.

Reply:
some changes to `poetry.lock` are to be expected from the update to Python 3.11, but none should be regarding Pyright or Pydantic
do you see the same errors when you run `pyright kpops/utils/gen_schema.py`? because I don't. Only with `pre-commit run pyright -a`

Reply:
If I run the command `pre-commit run --all-files` I get the errors